.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -3,15 +3,31 @@ description: Tell us about something that's not working the way we (probably) in
 labels: ["Platform: Cocoa", "Type: Bug"]
 body:
   - type: dropdown
-    id: environment
+    id: platform
```

Comment:
I have no clue why this was called environment before. I think it was a mistake.